modules/runner/src/test/java/org/apache/ignite/internal/configuration/compatibility/framework/ConfigurationTreeComparatorSelfTest.java

```diff
@@ -410,7 +448,7 @@ void testDeleteAfterRename() {
                 EnumSet.of(Flags.IS_ROOT));
 
         ConfigNode node1Ver1 = new ConfigNode(root, Map.of(Attributes.NAME, "oldTestCount"), List.of(),
-                EnumSet.of(Flags.IS_VALUE));
+                EnumSet.of(Flags.IS_VALUE, Flags.HAS_DEFAULT));
```

Comment:
is it really need for ("oldTestCount" and "newTestCount") to have Flags.HAS_DEFAULT ?

Reply:
seems this test need to be extended